Cython/Compiler/PyrexTypes.py

```diff
@@ -59,7 +59,8 @@ def specialization_name(self):
         if self._specialization_name is None:
             # This is not entirely robust.
             common_subs = (self.empty_declaration_code()
-                           .replace("unsigned ", "unsigned_")
+                           # covers both "unsigned " amd "signed "
+                           .replace("signed ", "signed_")
```

Comment:
amd -> and

Reply:
Thanks. This a fairly small PR to manage to fit two typos into